schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -37,6 +37,7 @@
                             },
                             {
                                 "oneOf": [
+                                    { "$ref": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.KeyVault.json#/definitions/secrets" },
```

Comment:
can you please make these change in 2014-04-01-preview/deploymentTemplate.json too?